PorticoExportDom.php

```diff
@@ -35,61 +36,49 @@ class PorticoExportDom
     /** @var string DTD ID of the exported XML */
     private const PUBMED_DTD_ID = '-//NLM//DTD JATS (Z39.96) Journal Archiving and Interchange DTD v1.2 20190208//EN';
```

Comment:
Off topic: Looks like the current version is JATS 1.4, of course it's not interesting to handle it here, but I think it would be useful to raise an issue in the `jatsTemplate`, to either move to this standard or to provide a way to generate other formats (the best for users and the worst for us).